qml/components/chatInformationPage/ChatInformationPageContent.qml

```diff
@@ -0,0 +1,459 @@
+/*
+    Copyright (C) 2020 Sebastian J. Wolf and other contributors
+
+    This file is part of Fernschreiber.
+
+    Fernschreiber is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Fernschreiber is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Fernschreiber. If not, see <http://www.gnu.org/licenses/>.
+*/
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import "./chatInformationPage"
+import "../js/twemoji.js" as Emoji
+import "../js/functions.js" as Functions
+
+
+SilicaFlickable {
+    id: pageContent
+    property alias membersList: membersList
+
+    function initializePage() {
+        console.log("[ChatInformationPage] Initializing chat info page...");
```

Comment:
I suggest to not leave console logs in files committed to master

Reply:
Perhaps we should think about log output in general (also c++) in the future. 
Only enable in debug builds? Command line parameter? 
But that's a topic for another place. Will remove here.

Reply:
That's one of the reasons why I'm turning those into macros on C++ size - to make it easier to manage those, even strip them completely at compile time.